.github/pull_request_template.md

```diff
@@ -1,4 +1,12 @@
+## Proposed changes
 
-Before submitting this PR, check the [contribution guidelines](CONTRIBUTING.md).
+Please include a description of the problem or feature this PR is addressing.
 
-Make sure your code is formatted: `pre-commit run --all-files`.
+## Checklist
+
+Put an `x` in the boxes that apply.
+
+- [ ] I have read the [CONTRIBUTING](https://github.com/ml-explore/mlx/blob/main/CONTRIBUTING.md) document
+- [ ] I have run `pre-commit run --all-files` to format my code
```

Comment:
Could you change this to "I have run `pre-commit run --all-files` to format my code / installed pre-commit prior to committing changes"